lib/ain-grpc/src/block.rs

```diff
@@ -72,8 +74,8 @@ impl RpcBlock {
                 }
             },
             uncles: vec![],
-            nonce: U256::zero(),
-            extra_data: block.header.extra_data,
+            nonce: block.header.nonce,
+            extra_data: Bytes(block.header.extra_data.clone()),
```

Comment:
Don't think this clone is necessary. 

```suggestion
            extra_data: Bytes(block.header.extra_data),
```